README.md

```diff
@@ -1 +1,12 @@
-# viveport_sdk_unreal_sample
\ No newline at end of file
+# VIVEPORT SDK Unreal Sample
+
+## If you encountered with VIVEPORT DRM wrapper compatibility issue in your unreal content, you can follow this sample to enable VIVEPORT DRM SDK.
+
+* Download ViveportSDK Unreal. https://developer.viveport.com/documents/sdk/en/unrealengine.html
+* Download DRM_fix_Component.
+* Copy DRM_fix_Component\ViveportSDK folder to your Unreal project .\Plugins folder.
+* Change .\Plugins\ViveportSDK\Source\ViveportSDK\Classes\ViveportDRMComponent.h to your VIVEPORT ID and VIVEPORT KEY.
+* If .\Plugins\ViveportSDK\Binaries and .\Plugins\ViveportSDK\Intermediate, exist , place remove them. Then remove your Project's .\Binaries and your Project's .\Intermediate folder.
```

Comment:
If .\Plugins\ViveportSDK\Binaries and .\Plugins\ViveportSDK\Intermediate exist , **please** remove them. Then remove your Project's .\Binaries and your Project's .\Intermediate folder.